staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -460,6 +460,9 @@ func (td *typeDiscoverer) discoverStruct(thisNode *typeNode, fldPath *field.Path
 			if !validations.Empty() {
 				klog.V(5).InfoS("found field-attached validations", "n", validations.Len())
 				child.fieldValidations.Add(validations)
+				if len(validations.Variables) > 0 {
```

Comment:
How about when we find list and map key/val validations?

Reply:
oops, added